inc/command/class-command.php

```diff
@@ -188,14 +202,224 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 		return $this->run_command( $input, $output, 'vendor/bin/phpunit', $options );
 	}
 
+	/**
+	 * Runs Codecept with zero config by default.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int
+	 */
+	protected function codecept( InputInterface $input, OutputInterface $output ) {
+		$options = $input->getArgument( 'options' );
+		$module = $input->getOption( 'module' );
+		$run_headless_browser = $input->getOption( 'browser' );
+		$use_chassis = $input->getOption( 'chassis' );
+		$tests_folder = $module !== 'project' ? "altis/$module/tests" : 'tests';
```

Comment:
I'm not really keen on the `-m` module option, it feels like too much of internal tool this way and has no value outside of our team. Although our goal is to use this to write our own tests the interface should really be geared towards devs building projects on Altis IMO.

It'll be more flexible for everyone if we can pass a path, then for module testing we can run `composer dev-tools codecept run vendor/altis/module` in the Travis configs.

I think codeception accepts a path as an argument and `tests` is already the default directory it'll look in: https://codeception.com/docs/02-GettingStarted#Running-Tests

Reply:
Resolving this as we agreed it'll be picked up by the project-specific setup ticket.